### DISCUSSION TITLE
Welcome to E2M Discussions!

### DISCUSSION THREAD

**Discussion by Jing-yilin** [Jing-yilin/E2M#1] [Announcements]
🎉 Welcome to the E2M (Everything to Markdown) Project Discussion Board! 🎉

We are excited to have you here and eager to hear your thoughts, ideas, and feedback. This space is dedicated to fostering open communication and collaboration among our community members.

Feel free to discuss any of the following topics:

💡 Feature Requests: Have an idea for a new feature? Share it here! We are always looking for ways to improve E2M and your input is invaluable.
🐛 Bug Reports: Encountered an issue? Let us know! Please provide as much detail as possible, including steps to reproduce the problem, so we can address it promptly.
❓ Usage Questions: Need help with something? Ask away! Whether you're new to E2M or an experienced user, we're here to assist you.
💬 General Feedback: We welcome all kinds of feedback. If you have suggestions, comments, or general thoughts about the project, we'd love to hear them.
How to Participate
🤝 Be Respectful: Treat everyone with respect and kindness. We value diverse perspectives and constructive dialogue.
📌 Stay on Topic: Keep discussions relevant to E2M and related topics. If you have off-topic questions, consider using direct messages or finding a more appropriate forum.
📝 Provide Context: When discussing issues or suggesting features, provide as much context as possible. This helps everyone understand and engage more effectively.
Thank you for being a part of our community. Together, we can make E2M better for everyone!

Happy discussing! 🎊

**Comment by parthasarathydNU** [comment]
Hey, this seems to be an amazing idea. Trying to convert all sorts of inputs into LLM friendly formats. I am currently working on a project that will be using a lot of structured and unstructured data in the Bio field. Protein sequences and related data in the following formats:  FASTA, GFF, and XML files. 

Do you think this tool will be extended to handle other non text based formats or are you only looking at text based formats for now.

Here's the project link for reference. https://github.com/parthasarathydNU/protein-data-nlq
